.github/ISSUE_TEMPLATE/request-for-change.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Request for change
+about: Tell us about a feature that you're working on
+title: "[RFC]"
+labels: RFC
+assignees: ''
+
+---
+
+The Dotcom team would like to help you with your proposed change. Please provide us with a brief and concise explanation for what you're looking to do we will try to make the process easier with early feedback.
+
+**Who is responsible for completing this RFC process?**
+Once the RFC is raised, it is the responsibility of the _Dotcom team_ to complete this process
```

Comment:
```suggestion
Once the RFC is raised, it is the responsibility of the _Dotcom team_ to provide feedback and help you get your change to the finish line.
```

Maybe? I feel like it currently implies that we're going to be the ones executing the RFC!